.gitignore

```
@@ -1,3 +1,5 @@
-
+# .nx files
 .nx/installation
-.nx/cache
\ No newline at end of file
+.nx/cache
+.nx/installation/node_modules
+/node_modules
```

```suggestion
node_modules
```
Instead of doing `/node_modules` which will only target the `node_modules` directory from the root (because of the `/`) , change do just `node_modules` and it will target any directory with that name.

---

Nice! I think that's where I messed up in the previous project! I used the "/" and found I kept having to add "/example/node_modules" for different folders that contained them.